BUILD.gn

```diff
@@ -224,6 +224,30 @@ template("spvtools_vendor_table") {
   }
 }
 
+template("spvtools_vendor_table_local") {
+  assert(defined(invoker.name), "Need name in $target_name generation.")
+
+  action("spvtools_vendor_tables_" + target_name) {
+    script = "utils/generate_grammar_tables.py"
+
+    name = invoker.name
+    extinst_vendor_grammar =
+        "source/extinst.${name}.grammar.json"
+    extinst_file = "${target_gen_dir}/${name}.insts.inc"
+
+    args = [
+      "--extinst-vendor-grammar",
+      extinst_vendor_grammar,
```

Comment:
Adding a rebase to this line compiles on my end:
```
rebase_path(extinst_vendor_grammar, root_build_dir),
```

Reply:
Excellent! Thanks. That's what I was looking for. I will add it to my PR and retry.